### PR TITLE
content: add talks

### LINKED ISSUE #101: A page or a section with our presentions and other resources
@ydcjeff I would like to put somewhere the links on our previous public presentations and other resources like posters etc.
Here is the content:

- PyData Global 2021 Sprint slides: https://pytorch-ignite.github.io/pydata-global2021-slides/
- Python Bucaramanga Meetup: https://vfdev-5.github.io/pybucaramanga-pytorch-ignite-slides/
- PTCV21: https://vfdev-5.github.io/ptcv21-pytorch-ignite-slides/
- PTED2021: https://drive.google.com/file/d/1YXrkJIepPk_KltSG1ZfWRtA5IRgPFz_U/
- PTDC2019: https://drive.google.com/file/d/1bqIl-EM6GCCCoSixFZxhIbuF25F2qTZg
- PTDC2018: https://drive.google.com/file/d/1_2vzBJ0KeCjGv1srojMHiJRvceSVbVR5
- ...

It can be a new page or a section somewhere, I do not know yet where it would be the best place.

### PR DESCRIPTION
**Pull request by ydcjeff**
fix #101 

@vfdev-5

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -47,5 +47,7 @@ params:
           link: /about/contribution-guide/
         - text: Code of Conduct
           link: /about/coc/
+        - text: Talks
```
Comment (vfdev-5) [inline config.yaml:50]: @ydcjeff I propose to put "Talks" into "About", what do you think ?

Comment (ydcjeff) [reply config.yaml:50]: Sounds good.

**Comment by ydcjeff** [comment]
I embed the youtube video directly using hugo built-in shortcode, `youtube`. You can revert https://github.com/pytorch-ignite/pytorch-ignite.ai/pull/114/commits/061f1ebd0622c87d4cbc18c7974e82b20785f742 if you find youtube videos take much spaces.

**Review by vfdev-5 — APPROVED** [review]

**Comment by vfdev-5** [comment]
@ydcjeff yes, looks good. Do you think we can make them a bit smaller ?

**Comment by ydcjeff** [comment]
Hugo built-in shortcode doesn't provide options for re-sizing.

**Pull request merged**